reproductions/basic-sqlite/package.json

```diff
@@ -5,9 +5,9 @@
   "main": "index.ts",
   "scripts": {
     "dbpush": "prisma db push --skip-generate",
-    "generate": "prisma generate",
-    "test": "ts-node index.ts",
-    "debug": "node -r ts-node/register --inspect-brk index.ts"
+    "generate": "PRISMA_COPY_RUNTIME_SOURCEMAPS=1 prisma generate",
```

Comment:
I assume this works out of the box in functional tests because we link directly to the runtime file?

Reply:
Yes, and source map is always besides it. Also, `jest` uses `source-map-support` package by default, so there is no need to use `--enable-source-maps` flag.